CleanWeather/Scenes/CityDetails/CityDetailsCreator.swift

```diff
@@ -0,0 +1,28 @@
+//
+//  CityDetailsCreator.swift
+//  CleanWeather
+//
+//  Created by Marek Skrzelowski on 19/11/2019.
+//  Copyright © 2019 vandermesis. All rights reserved.
+//
+
+import Foundation
+
+struct CityDetailsCreator {
+    
+    func getController(with passedCityWeather: CityWeather) -> CityDetailsController {
+        
+        let passedData = passedCityWeather
+        let networking = WeatherNetworkingImpl()
+        let worker = CityDetailsWorkerImpl(networking: networking)
+        let router = CityDetailsRouterImpl()
+        let presenter = CityDetailsPresenterImpl()
+        let interactor = CityDetailsInteractorImpl(presenter: presenter, worker: worker, router: router, passedData: passedData)
```

Comment:
not passedData, but cityDetails

and something that is less important:
I'd better keep that kind of parameters as first, beforee presenter, worker and router